Package.swift

```diff
@@ -1,11 +1,22 @@
 // swift-tools-version:5.0
 import PackageDescription
 
+var platforms: [SupportedPlatform] {
+    #if compiler(<5.3)
+        return [
+            .macOS(.v10_10), .iOS(.v8), .tvOS(.v9), .watchOS(.v2)
+        ]
+    #else
+        // Xcode 12 (which ships with Swift 5.3) drops support for iOS 8
+        return [
+            .macOS(.v10_10), .iOS(.v9), .tvOS(.v9), .watchOS(.v2)
+        ]
+    #endif
+}
+
 let package = Package(
     name: "APIKit",
-    platforms: [
-        .macOS(.v10_10), .iOS(.v8), .tvOS(.v9), .watchOS(.v2)
-    ],
+    platforms: platforms,
```

Comment:
Thanks 👍 
However, iOS 8 is now rarely used, so we plan to end support for iOS 8 in the next major version. Can you simply change the minimum supported version of iOS to `v9`?

Reply:
Sounds good. 
I'll leave it to you that compatibility. 👍